tests/Common/ComponentsTest.php

```diff
@@ -1839,6 +1846,22 @@ public function testComponentConfigRowUpdate()
         $this->assertNotEmpty($version['created']);
         $this->assertEquals($newToken['id'], $version['creatorToken']['id']);
         $this->assertEquals($newToken['description'], $version['creatorToken']['description']);
+
+        $components->updateConfigurationRow(
+            $configurationRow
+                ->setName('Renamed Main 1')
+                ->setChangeDescription(null)
```

Comment:
Tady jsem dal null, který víc odpovídá tomu, co by tam bylo kdybych to nenastavil.